task 7

```diff
@@ -3,4 +3,4 @@ def task
     myArray = ["stuff", "widget", "ruby", "goodies", "java", "emerald", "etc" ]
```

Comment:
this script must take arguments from outside (`ARGV`), so you should give these argument when you call this script